src/metabase/driver/druid/query_processor.clj

```diff
@@ -520,14 +517,33 @@
         (apply dissoc result keys-to-remove)))))
 
 
-;;; ### process-mbql-query
-
-(defn process-mbql-query
-  "Process an MBQL (inner) query for a Druid DB."
-  [do-query query]
-  (binding [*query* query]
-    (let [[query-type druid-query] (build-druid-query query)]
-      (log-druid-query druid-query)
```

Comment:
All native forms would be logged in the QP `run-query` function so we don't need individualized logging functions.